src/app/components/publiclists/teamlist/teamcard/teamcard.component.html

```diff
@@ -1,41 +1,40 @@
 <div class="team-card-container" [class.selected]='isSelected' [class.noexpand] = 'isHost' [class.expand] = '!isHost'>
 <div class="team-card-container-child">
-  <div class="team-content" (click)='selectToggle()'>
-    <i class="fa fa-circle selectstate" *ngIf='!isSelected && (isHost || isOnChallengePage)'></i>
-    <i class="fa fa-check selectstate" aria-hidden="true" *ngIf='isSelected && (isHost || isOnChallengePage)'></i>
-    <i class="fa fa-users" aria-hidden="true"></i>
-    <div class="team-details">
-      <span class="title toptitle">
-	    Team:
-      </span>
-      <span *ngIf="teamView['team_url']">
-      	<a class="orange-text" target="_blank" href="{{teamView['team_url']}}">{{teamView['team_name']}}</a>
-      </span>
-      <span *ngIf="!teamView['team_url']">
-	    {{teamView['team_name']}}
-      </span>
-      <br>
-      <span class="title">
-	    Created by:
-      </span>
-      <span>
-	    {{teamView['created_by']}}
-      </span>
+    <div class="team-content" (click)='selectToggle()'>
+      <input type="radio" class="with-gap selectPhase" name="TeamSelection" (click)='selectTeamToggle()' id="{{teamView['team_name']}}">
+      <label for="{{teamView['team_name']}}"></label>
+      <i class="fa fa-users" aria-hidden="true"></i>
+      <div class="team-details">
+        <span class="title toptitle">
+        Team:
+        </span>
+        <span *ngIf="teamView['team_url']">
+          <a class="orange-text" target="_blank" href="{{teamView['team_url']}}">{{teamView['team_name']}}</a>
+        </span>
+        <span class="orange-text" *ngIf="!teamView['team_url']">
+        {{teamView['team_name']}}
+        </span>
+        <br>
+        <span class="title">
+        Created by:
+        </span>
+        <span>
+        {{teamView['created_by']}}
+        </span>
+      </div>
+      <div class='team-icons'>
+        <a (click)="addMembersToTeam($event)"><i class="fa fa-user-plus list-icon text-highlight w-300 left" aria-hidden="true"></i></a>
+        <a (click)="editTeam($event)"><i class="fa fa fa-pencil list-icon text-highlight w-300 left grey-text" aria-hidden="true"></i></a>
+        <a (click)="deleteTeam($event)"><i class="fa fa-trash list-icon text-highlight w-300 left grey-text" aria-hidden="true"></i></a>
```

Comment:
Minor: Can you please do the same spacing as in the current version?

Reply:
Sir  space is same as current version